src/test/suite/tokenizer.test.ts

```diff
@@ -1,8 +1,15 @@
 import * as assert from "assert";
 import { tokenize } from "../../core/tokenizer";
 import { flatten, range } from "lodash";
+import { SupportedLanguageId } from "../../languages/constants";
 
 type TestCase = [string, string[]];
+type LanguageTokenizerTests = Partial<
+  Record<
+    SupportedLanguageId,
+    { tests: TestCase[]; exclusionPredicate?: (input: string) => boolean }
+  >
+>;
 const singleSymbolTests: TestCase[] = getAsciiSymbols().map((s) => [s, [s]]);
 
 const tests: TestCase[] = [
```

Comment:
Maybe let's rename this one so the shadowing thing doesn't happen again
```suggestion
/** Tokenizer tests that will be run on the default tokenizer, as well as on all language-specific tokenizers */
const globalTests: TestCase[] = [
```